support/do.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+useradd --create-home --shell /bin/bash --user-group deployer
+cp -r /root/.ssh /home/deployer
+chown -R deployer:deployer /home/deployer/.ssh 
+echo 'deployer ALL=(ALL:ALL) ALL' >> /etc/sudoers
```

Comment:
What do you propose then? Should we have a `provisioner` user with sudo privedges? Or should we whitelist the commands available to `deployer`?